easybuild/easyblocks/c/clang.py

```diff
@@ -73,6 +74,13 @@
 # List of all supported CUDA toolkit versions supported by LLVM
 CUDA_TOOLKIT_SUPPORT = ['80', '90', '91', '92', '100', '101', '102', '110', '111', '112']
 
+# List of the known LLVM projects
+KNOWN_LLVM_PROJECTS = ['llvm', 'compiler-rt', 'clang', 'openmp', 'polly', 'lld', 'lldb',
+                       'clang-tools-extra', 'flang']
+
+# List of the known LLVM runtimes
+KNOWN_LLVM_RUNTIMES = ['libunwind', 'libcxx', 'libcxxabi']
```

Comment:
So the full list of runtimes is actually longer, see https://llvm.org/docs/GettingStarted.html#local-llvm-configuration and this is "Only effective when using the full monorepo layout" (which I'm guessing means the single source tarball). I'll make a PR to your easyblock to support this.

Reply:
PR opened at https://github.com/ComputeCanada/easybuild-easyblocks/pull/15

Reply:
I saw this, but I noticed that only `libunwind`, `libcxx` and `libcxxabi` end up in the runtime directory, so I assumed this was not yet transitioned.

Reply:
i.e.: 
```
ls /home/mboisson/.local/easybuild/software/2020/avx2/Core/clang/15.0.2/lib/x86_64-unknown-linux-gnu/
libc++.a  libc++abi.a  libc++abi.so  libc++abi.so.1  libc++abi.so.1.0  libc++experimental.a  libc++.so  libc++.so.1  libc++.so.1.0  libunwind.a  libunwind.so  libunwind.so.1  libunwind.so.1.0
```